### PR TITLE
SDKS-744 Do not call call /endSession when the access token is bound …

### PR DESCRIPTION
**Pull request by witrisna**
Do not call call /endSession when the access token is bound to a session

### CODE REVIEW

File: forgerock-auth/src/test/java/org/forgerock/android/auth/FRUserMockTest.java [hunk 1]
```diff
@@ -973,7 +961,7 @@ public void testRequestInterceptor() throws InterruptedException, ExecutionExcep
         Assertions.assertThat(result.get("EXCHANGE_TOKEN").second).isEqualTo(1);
         Assertions.assertThat(result.get("REVOKE_TOKEN").second).isEqualTo(1);
         Assertions.assertThat(result.get("LOGOUT").second).isEqualTo(1);
-        Assertions.assertThat(result.get("END_SESSION").second).isEqualTo(1);
+        //Assertions.assertThat(result.get("END_SESSION").second).isEqualTo(1);
```
Comment (rodrigoareis) [inline forgerock-auth/src/test/java/org/forgerock/android/auth/FRUserMockTest.java:964]: Is this comment intentional or left here accidentally ?

**Review by rodrigoareis — DISMISSED** [review]
This commit looks good to me

File: forgerock-auth/src/test/java/org/forgerock/android/auth/FRUserMockTest.java (hunk 1 above)
Comment (witrisna) [reply forgerock-auth/src/test/java/org/forgerock/android/auth/FRUserMockTest.java:964]: Good Catch, that is intentional, but the END_SESSION interceptor test is missing. So I have added a test case to test END_SESSION action.

**Review by rodrigoareis — APPROVED** [review]
Changes looks good to me

**Pull request merged**